example/README.md

```diff
@@ -0,0 +1,87 @@
+# Overview
+This directory contains a simple open source example/demo that we can provide to any user who wants to learn how concord-bft works and how they may use it. 
+
+This example is responsible for assisting and guiding many concord-bft users throughout the world, including those who want to understand more about concord-bft, its usage, and how it works. This open source example/demo will provide end to end functionality of Concord byzantine fault tolerant state machine replication library about its usage and how we can use this as a library to implement a highly scalable and energy-efficient distributed trust infrastructure for consensus and smart contract execution.
+So far, we're intending to implement this using an open source replica kvbc, similar to how we do it in Apollo tests, and then, after the setup is complete, we'll include other open source execution engines for this showcase, such as Ethereum, WASM, etc.
```

Comment:
This sounds like a historical reference, rather than a description of the current project. I think you should rather emphasize on the fact that:

- this example is a byzantine fault tolerant K/V-store implementation, based on Concord-BFT consensus
- it is a fully wired system, that is self-service and ready for demoing
- other important topics to highlight @itsmeakashgoyal and @arc-vmware ?